service-tests/ansible.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'ansible', title: 'Ansible Galaxy' });
+module.exports = t;
+
+t.create('ansible role name')
+ .get('/role/14542.json')
+ .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('role'),
+    value: Joi.equal('openwisp.openwisp2')
+ }));
+
+t.create('ansible role download counts')
+ .get('/role-downloads/14542.json')
+ .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('role downloads'),
+    value: Joi.number().integer()
```

Comment:
This passes for now, yet it seems too restrictive, since it won't match e.g. **1.1k**.

We should have a standard regex for the output of our metric function. This one from `service-tests/conda.js` looks good. It could be moved into `service-tests/runner/helpers.js` and imported from there, so contributors don't have to keep reinventing the wheel.

```js
Joi.string().regex(/^[0-9]+[kMG]?$/)
```

Reply:
will try it now